test/test_helper.rb

```diff
@@ -17,10 +18,14 @@ class ActiveSupport::TestCase
   include FactoryBot::Syntax::Methods
   include GemHelpers
   include EmailHelpers
+  include PasswordHelpers
 
   setup do
     I18n.locale = :en
     Rails.cache.clear
+
+    # Don't connect to the Pwned Passwords API in tests
+    Pwned.stubs(:pwned?).returns(false)
```

Comment:
Does this mean other then API tests do need to connect to Pwned Passwords API?

Reply:
Nothing in the test suite should make actual HTTP calls to Pwned Passwords, and based on my testing this is the case.

In the test suite passwords are only validated against the bloom filter included with the Unpwn gem.

Reply:
ok, thx for info